### DISCUSSION TITLE
Problem with a version of the pulumi-github package

### DISCUSSION THREAD

**Discussion by yvanegas** [pulumi/pulumi#13952] [Pulumi Packages]
How can I solve the following problem?

`Diagnostics:
  pulumi:pulumi:Stack (****):
    could not create provider urn:pulumi:qa::****::pulumi:providers:github::default_4_11_0_alpha_1645193352::fadcb75f-aad6-4761-acbc-0d521a76d355: load plugin for github provider 'urn:pulumi:qa::****::pulumi:providers:github::default_4_11_0_alpha_1645193352': Could not automatically download and install resource plugin 'pulumi-resource-github' at version v4.11.0-alpha.1645193352, install the plugin using `pulumi plugin install resource github v4.11.0-alpha.1645193352`: error downloading provider github to file: failed to download plugin: github-4.11.0-alpha.1645193352: 403 HTTP error fetching plugin from https://get.pulumi.com/releases/plugins/pulumi-resource-github-v4.11.0-alpha.1645193352-linux-amd64.tar.gz`

**Comment by Frassle** [comment]
> How can I solve the following problem?

Don't use an alpha version of the github provider. The latest version is [v5.17.0](https://github.com/pulumi/pulumi-github/releases/tag/v5.17.0), or use the closer [v4.11.0](https://github.com/pulumi/pulumi-github/releases/tag/v4.11.0).

**Comment by ringods** [comment]
@yvanegas I have a few questions:

* is this an existing stack?
* what programming language are you using for this stack?
* which version of Pulumi's `github` package SDK are you using in that setup?
* Is this the same alpha version?

**Reply by yvanegas** [reply]
- Yes, it is an existing stack!
- Python
- I am not using the SDK

**Reply by Frassle** [reply]
> I am not using the SDK

That will probably be the issue then. This program must have had github resources in it that are in the statefile, but the resources have been removed from the program so the engine wants to delete them all and it will use the provider it knows about to do that.

Best choice here is quite possibly to find the problematic resources with `pulumi stack --show-urns` and just remove them all from state with `pulumi state delete`.
